src/main/scala/io/citrine/lolo/validation/Merit.scala

```diff
@@ -132,6 +134,64 @@ case object UncertaintyCorrelation extends Merit[Double] {
   }
 }
 
+/**
+  * Negative Log Probability Density (NLPD) in two dimensions
+  * NLPD is calculated for each point and the median value is returned.
+  *
+  * @param i index of the first label to be compared
+  * @param j index of the second label to be compared
+  * @param method method to calculate correlation coefficient
+  * @param observational whether or not to calculate the observational uncertainty
+  */
+case class NegativeLogProbabilityDensity2d(i: Int, j: Int, method: CorrelationMethod, observational: Boolean) extends Merit[Seq[Any]] {
+  override def evaluate(predictionResult: PredictionResult[Seq[Any]], actual: Seq[Seq[Any]], rng: Random): Double = {
+    val pvas = PredictedVsActualTwoDimensions.makePva2d(predictionResult, actual, i, j, method, observational)
+    val nlpd = NLPD(pvas)
+    StatsUtils.median(nlpd)
```

Comment:
Would it be useful to look at max(NLPD) to try to get a sense of how bad the "worst" prediction is in terms of how far the actual falls outside the predicted uncertainty region? 

Or is this accounted for by the PVA2D metric that uses a cutoff on the Mahalaonbis distance?

Reply:
I was thinking that the impact of a poor prediction levels off after a certain point. That is, being off by 10 standard deviations is really bad. Being off by 20 standard deviations is basically the same amount of really bad.

> Or is this accounted for by the PVA2D metric that uses a cutoff on the Mahalaonbis distance?

Are you referring to standard confidence? No. That measures how many predictions are within 1 (generalized) standard deviation. Being off by 10 standard deviations is the same as being off by 1.5.